### PR TITLE
Update manage-deployment-of-add-ins.md

### PR DESCRIPTION
**Pull request by msbemba**
Added steps to Edit the user access and delete add-in if need be.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/manage/manage-deployment-of-add-ins.md [hunk 1]
```diff
@@ -130,6 +130,16 @@ Updates for add-ins happen as follows:
 - **Line-of-business add-in:** In this case, where an admin explicitly uploaded a manifest, the add-in requires that the admin upload a new manifest file to support metadata changes. The next time the relevant Office applications start, the add-in will update. The web application can change at any time. 
     
 - **Office Store add-in:** When an admin selected an add-in from the Office Store, if an add-in updates in the Office Store, the add-in will update later in Centralized Deployment. The next time the relevant Office applications start, the add-in will update. The web application can change at any time. 
+
+### Edit Add-in access
+
+Post Deployment, Admin can also modify the user access to add-in
```
Comment (JohanFreelancer9) [inline Office365-Admin/manage/manage-deployment-of-add-ins.md]:
```suggestion
Post Deployment, Admin can also modify the user access to add-ins.
```

File: Office365-Admin/manage/manage-deployment-of-add-ins.md [hunk 2]
```diff
@@ -130,6 +130,16 @@ Updates for add-ins happen as follows:
 - **Line-of-business add-in:** In this case, where an admin explicitly uploaded a manifest, the add-in requires that the admin upload a new manifest file to support metadata changes. The next time the relevant Office applications start, the add-in will update. The web application can change at any time. 
     
 - **Office Store add-in:** When an admin selected an add-in from the Office Store, if an add-in updates in the Office Store, the add-in will update later in Centralized Deployment. The next time the relevant Office applications start, the add-in will update. The web application can change at any time. 
+
+### Edit Add-in access
+
+Post Deployment, Admin can also modify the user access to add-in 
+
+1. In the admin center, go to the Settings > Services & add-ins page.
```
Comment (JohanFreelancer9) [inline Office365-Admin/manage/manage-deployment-of-add-ins.md]:
```suggestion
1. In the admin center, go to the **Settings** > **Services & add-ins** page.
```

File: Office365-Admin/manage/manage-deployment-of-add-ins.md [hunk 3]
```diff
@@ -130,6 +130,16 @@ Updates for add-ins happen as follows:
...
+
+2. Select the Deployed Add-in.
```
Comment (JohanFreelancer9) [inline Office365-Admin/manage/manage-deployment-of-add-ins.md]:
```suggestion
2. Select the deployed add-in.
```

File: Office365-Admin/manage/manage-deployment-of-add-ins.md [hunk 4]
```diff
@@ -130,6 +130,16 @@ Updates for add-ins happen as follows:
...
+
+3. Click on **Edit** under **Who has Access**
```
Comment (JohanFreelancer9) [inline Office365-Admin/manage/manage-deployment-of-add-ins.md]:
```suggestion
3. Click on **Edit** under **Who has Access**.
```

File: Office365-Admin/manage/manage-deployment-of-add-ins.md [hunk 5]
```diff
@@ -251,6 +261,16 @@ If the deployed add-in doesn't support add-in commands or if you want to view al
 1. On the **Home** ribbon, select **Get Add-ins**.<br/>![Store button in Outlook](../media/getaddinsicon.png)
   
 2. Select **Admin-managed** in the left nav.
+
+## Delete the add-in
+
+If you want to delete the add-in that was deployed
```
Comment (JohanFreelancer9) [inline Office365-Admin/manage/manage-deployment-of-add-ins.md]:
```suggestion
You can also delete an add-in that was deployed.
```

File: Office365-Admin/manage/manage-deployment-of-add-ins.md [hunk 6]
```diff
@@ -251,6 +261,16 @@ If the deployed add-in doesn't support add-in commands or if you want to view al
...
+
+1. In the admin center, go to the Settings > Services & add-ins page.
```
Comment (JohanFreelancer9) [inline Office365-Admin/manage/manage-deployment-of-add-ins.md]:
```suggestion
1. In the admin center, go to the **Settings** > **Services & add-ins** page.
```

File: Office365-Admin/manage/manage-deployment-of-add-ins.md [hunk 7]
```diff
@@ -251,6 +261,16 @@ If the deployed add-in doesn't support add-in commands or if you want to view al
...
+
+2. Select the Deployed Add-in.
```
Comment (JohanFreelancer9) [inline Office365-Admin/manage/manage-deployment-of-add-ins.md]:
```suggestion
2. Select the deployed add-in.
```

File: Office365-Admin/manage/manage-deployment-of-add-ins.md [hunk 8]
```diff
@@ -251,6 +261,16 @@ If the deployed add-in doesn't support add-in commands or if you want to view al
...
+
+3. Click on **Delete Add-In**
```
Comment (JohanFreelancer9) [inline Office365-Admin/manage/manage-deployment-of-add-ins.md]:
```suggestion
3. Click on **Delete Add-In**.
```

**Comment by mypil** [comment]
@msbemba  - please merge the commit on the copy/edit review done by JohanFreelancer9  or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@Sirkkuw - PR has been copy edited and is ready for final review. This is based on issue #826 

Thank you.

cc: @kenwith

**Pull request merged**